code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -1,3 +1,7 @@
+#define MUTE_NONE 0 // Does not mute you.

```

Comment:
These defines should be moved to one of the `__DEFINES\` files and used in the place you check `mute == 2` and `mute == 1`.